package.json

```
@@ -38,7 +41,8 @@
     "pretest-php-multisite": "wp-env run composer 'install --no-interaction'",
     "test-php-multisite": "wp-env run phpunit 'WP_MULTISITE=1 phpunit -c /var/www/html/wp-content/plugins/$(basename $(pwd))/tests/multisite.xml --verbose'",
     "wp-env": "wp-env",
-    "prepare": "husky install"
+    "prepare": "husky install && npm run copy-vendor-libs",
```

If we don't need the vendor libs for development, then this wouldn't be needed here. It could rather be run as part of `build-plugins` above.

```suggestion
    "prepare": "husky install",
```